appuio/haproxy/README.gotmpl.md

```diff
@@ -134,17 +134,23 @@ Set `haproxy.config` to `redisk8s` to use the Redis configuration with DNS servi
 | `haproxy.redisk8s.metrics.enabled`| If the metric endpoint of the Redis backends should be exposed in haproxy | `false`
 | `haproxy.redisk8s.metrics.exposeLoadbalancer`| If the metric endpoint of the Redis backends should be exposed in the haproxy service | `true`
 
+### filterproxy
+
+| Parameter              | Description            | Default
+|---                     | ---                    | ---
+| `filterproxy.enabled` | Enables exposing an [exporter-filterproxy](https://github.com/vshn/exporter-filterproxy) that will be filtered based on the namespace | `false`
+| `filterproxy.url` | The URL of the exporter-filterproxy to expose | `""`
 
 ## Galera and Redis metrics
 
 By setting the `haproxy.galerak8s.metrics.enabled` or `haproxy.redisk8s.metrics.enabled` parameters, you can expose the metrics of the backend pods.
 
 For Galera the endpoints are:
-* `<haproxy_service_ip>:9090/metrics/mariadb-0/metrics`
-* `<haproxy_service_ip>:9090/metrics/mariadb-1/metrics`
+* `<haproxy_service_ip>:9090/mariadb/0`
+* `<haproxy_service_ip>:9090/mariadb/1`
 * `...`
 
 For Redis the endpoints are:
-* `<haproxy_service_ip>:9090/metrics/redis-0/metrics`
-* `<haproxy_service_ip>:9090/metrics/redis-1/metrics`
+* `<haproxy_service_ip>:9090/redis/0`
+* `<haproxy_service_ip>:9090/redis/1`
```

Comment:
The new paths are correct for the filterproxy implementation. The regular metrics path would actually be `<haproxy_service_ip>:9090/metrics` with a host header of `redis-0`, `redis-1`, etc.

```
curl -H "Host: redis-0" haproxy:9090/metrics
```

Reply:
You're right this doesn't explain the whole picture (and before the documentation was completely wrong though) I'll improve it